app/types/settings.ts

```diff
@@ -2,7 +2,7 @@ export interface UiSettings {
   romaji: boolean
   kanji: boolean
   furigana: boolean
-  translation: boolean
+  translation: boolean | string
```

Comment:
I guess it's easier to have this a seperate prop `translationLocale` to make it easier, and preserve the last choice when toggling the translations display